x-pack/plugins/apm/server/lib/rum_client/get_web_core_vitals.ts

```diff
@@ -132,7 +132,7 @@ export async function getWebCoreVitals({
 
   return {
     coreVitalPages: coreVitalPages?.doc_count ?? 0,
-    cls: cls?.values[pkey]?.toFixed(3) || null,
+    cls: cls?.values[pkey] ?? null,
```

Comment:
Because cls is required in the type `UXMetrics`, and defined as `number | null`, we need to default to `null` in the case where cls is `undefined` in order to satisfy the `UXMetrics` type.